platform/ios/resources/ja.lproj/Localizable.strings

```diff
@@ -43,6 +43,9 @@
 /* Action sheet title */
 "SDK_NAME" = "Mapbox iOS SDK";
 
+/* Developer-only SDK update notification; {latest version, in format x.x.x} */
+"SDK_UPDATE_AVAILABLE" = "現在Mapbox iOS SDK %1$@が入手できる：";
```

Comment:
In general, you should wait until Transifex notices the new base string to translate it over there, rather than checking the translation in ahead of time.